soc/arm/nxp_imx/rt/Kconfig.soc

```diff
@@ -9,6 +9,9 @@ choice
 
 config SOC_MIMXRT1011
 	bool "SOC_MIMXRT1011"
+	select CPU_CORTEX_M7
```

Comment:
should we put CPU_CORTEX_M7 and CPU_CORTEX_M_HAS_DWT under SOC_SERIES_IMX_RT10XX instead of duplicating them?

Reply:
Thanks, I will make this change.